### DISCUSSION TITLE
Collapsing an accordion panel

### DISCUSSION THREAD

**Discussion by jbvouma** [mantinedev/mantine#3986] [Q&A] [answered]
Hi,

I was wondering if there is a way to trigger a collapse of an accordion with a button click.

In the example below, I would like to collapse the accordion Panel on apply button click.

![image](https://user-images.githubusercontent.com/67848363/229594912-b05fc102-fdb0-4ff0-99c9-65e2d8422f79.png)

I checked the doc and didn't find anything. I know there is a collapse component but it doesn't really apply to my case here.

Thanks!

**Comment by rtivital** [comment, marked as answer]
You can control opened accordion items state – https://mantine.dev/core/accordion/#control-state

**Reply by jbvouma** [reply]
🤦‍♂️ Missed this part in the doc. Thanks!
